cli/command/cli_options.go

```diff
@@ -108,3 +113,107 @@ func WithAPIClient(c client.APIClient) CLIOption {
 		return nil
 	}
 }
+
+// envOverrideHTTPHeaders is the name of the environment variable that can be
```

Comment:
nit: a bit pedantic, but should we go with only one way of writing these in the docstrings?
- `environment variable`
- `environment-variable`
- `env-var`

Reply:
Heh, yeah, I keep making that mistake. Changed to `environment-variable` which seems the most common use, but we should make a pass at looking through our use for consistency